docs/logs/discovery/tailing-k8s-logs.md

```diff
@@ -0,0 +1,24 @@
+title: Shipping Kubernetes Logs
+description: Shipping Kubernetes Log Sources
+
+Collecting and shipping Kubernetes logs is accomplished in a similar manner to [log files](/logs/discovery/tailing-log-files) and [plain container](/logs/discovery/tailing-containers) logs. The Kubernetes containers are grouped by type of service or process that is running inside the container.
+
+![Kubernetes Logs Discovery](images/kubernetes-logs-discovery.png)
+
+To start shipping the logs that are produced by the specific service group or process, click on the `Set Up` button.
+
+![Kubernetes Log Shipping](images/k8s-setup-log-shipping.png)
+
+This will bring in the following flyout panel where you can configure Kubernetes log shipping:
```

Comment:
This will bring in the following flyout panel where you can configure Kubernetes log shipping
=>
This will display a panel where you can configure Kubernetes log shipping: